Compilation error

Compilation of latest netdata failing on Ubuntu 18.04 with cmake (via CLion)
```
CMakeFiles/netdata.dir/daemon/main.c.o: In function `get_system_info':
/home/chris/CLionProjects/netdata/daemon/main.c:768: undefined reference to `coverity_remove_taint'
/home/chris/CLionProjects/netdata/daemon/main.c:769: undefined reference to `coverity_remove_taint'
collect2: error: ld returned 1 exit status
CMakeFiles/netdata.dir/build.make:1974: recipe for target 'netdata' failed
make[3]: *** [netdata] Error 1
CMakeFiles/Makefile2:87: recipe for target 'CMakeFiles/netdata.dir/all' failed
make[2]: *** [CMakeFiles/netdata.dir/all] Error 2
CMakeFiles/Makefile2:94: recipe for target 'CMakeFiles/netdata.dir/rule' failed
make[1]: *** [CMakeFiles/netdata.dir/rule] Error 2
Makefile:118: recipe for target 'netdata' failed
make: *** [netdata] Error 2
```

--- a/daemon/main.c
+++ b/daemon/main.c
@@ -732,7 +732,7 @@ static int load_netdata_conf(char *filename, char overwrite_used) {
 }
 
 // coverity[ +tainted_string_sanitize_content : arg-0 ]
-inline void coverity_remove_taint(char *s)
+static inline void coverity_remove_taint(char *s)
 {
     (void)s;
 }

